src/lfc_scripts.py

```diff
@@ -1,13 +1,61 @@
 import mwclient
 import config
+from geopy.geocoders import Nominatim
+import folium
 
-site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
-site.login(config.username, config.api_key)
+def main():
+    site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
+    site.login(config.username, config.api_key)
 
-response = site.api(
-    'torquedataconnect',
-    format='json',
-    path='/competitions/LoneStar2020/proposals'
-)
+    response = site.api(
+        'torquedataconnect',
+        format='json',
+        path='/competitions/LoneStar2020/proposals'
+    )
+    proposal_ids = response["result"]
 
-print(response["result"]) 
\ No newline at end of file
+    geolocator = Nominatim(user_agent = "map")
+
+    #Extract organization locations
+    locations = []
+    for id in proposal_ids:
+
+        response = site.api(
+            'torquedataconnect',
+            format='json',
+            path='/competitions/LoneStar2020/proposals' + '/' + id
+        )
+        response['result']
+        org_location = geolocator.geocode(concat_org_location(response['result'])) #Convert named address to latlong pair
+        try:
+            coordinate_pair = (org_location.latitude, org_location.longitude)
+            print(org_location.latitude, org_location.longitude)
+        except:
+            print('error, not a location')
+        locations.append(coordinate_pair)
+
+    #Plot latlong points
+    mapit = folium.Map(location = [48, -102], zoom_start=6)
+    for coord in locations:
+        folium.Marker(location=[coord[0], coord[1]], fill_color='#43d9de', radius=4).add_to(mapit)
+    mapit.save('map.html')
+
+def concat_org_location(proposal):
+
+    org_location = []
+    for key, value in proposal.items():
+        if key == "Org Locality/ District/ County":
+            org_location.append(value)
+        if key == "Org State/Province":
+            org_location.append(value)
+        if key == "Org Country":
+            org_location.append(value)
+    return org_location
+
+'''
+def concat_future_work_location(proposal):
```

Comment:
It looks like these lines may be for future work.  Let's not include them in this PR.

Reply:
Removed